plugins/com.google.cloud.tools.eclipse.sdk/src/com/google/cloud/tools/eclipse/sdk/internal/PreferenceConstants.java

```diff
@@ -28,10 +28,26 @@
   // host bundle for the preference
   static final String BUNDLEID = "com.google.cloud.tools.eclipse.sdk";
 
+  public static enum CloudSdkManagement {
+    MANAGED, CUSTOM
+  }
+
+  /**
+   * Preference name for how the Google Cloud SDK is managed.
+   *
+   * @see CloudSdkManagement
```

Comment:
Should be clear what the acceptable preference values are? `@see CloudSdkManagement#name()`?